### PR TITLE
Review Comments Incorporated

### PR DESCRIPTION
**Pull request by srepking**

### CODE REVIEW

File: students/srepking/Lesson09/donors.py [hunk 1]
```diff
@@ -78,10 +72,12 @@ def column_number_width(donor_summary):
...
     @staticmethod
     def sort_list(donor_summary):
-        list_sorted = sorted(donor_summary, key=donor_summary.__getitem__, reverse=True)
+        list_sorted = sorted(donor_summary,
+                             key=donor_summary.__getitem__, reverse=True)
         return list_sorted
 
-    def make_report(self):
+    @property
+    def report(self):
```
Comment (natasha-aleksandrova) [inline students/srepking/Lesson09/donors.py]: `report` is more of a function than a property.

File: students/srepking/Lesson09/donors.py [hunk 2]
```diff
@@ -122,19 +116,15 @@ def letters(self):
...
-
-
-# Method to add a new donor
-
-# Method to search for a given donor
-
-
-# Generate reports about multiple donors
\ No newline at end of file
+                .format(self.donations[-1], self.name))
```
Comment (natasha-aleksandrova) [inline students/srepking/Lesson09/donors.py]: when creating objects try to think of all of the properties and features that relate to the object; in this scenario what else belong to single Donor?
* avg donation
* total donations
* sum donations

those can be functions (or even properties if you must)

File: students/srepking/Lesson09/donors.py [hunk 3]
```diff
@@ -104,15 +100,13 @@ def make_report(self):
...
 
-        for donor, donation in self._donor_raw.items():
+        for donor, donor_obj in self._donor_raw.items():
             donation_summary = donors_f[donor]
             letter = f'Dear {donor}, thank you so much for your ' \
-                     f'last contribution of ${donation[-1]:.2f}! ' \
+                     f'last contribution of ${donor_obj.donations[-1]:.2f}! ' \
                      f'You have contributed a total of $' \
                      f'{donation_summary[0]:.2f}, ' \
                      f'and we appreciate your support!'
```
Comment (natasha-aleksandrova) [inline students/srepking/Lesson09/donors.py]: this letter text could also live in the Donor class

**Comment by srepking** [comment]
Hi Natasha, 
I incorporated these comments. Thank you for them!

**Pull request merged**